openedx/features/edly/utils.py

```diff
@@ -0,0 +1,18 @@
+from openedx.features.edly.models import EdlyUserProfile
+
+
+def user_has_edly_organization_access(request):
+    """
+    Return True if the edly-sub-organization related to request URL is listed in get_linked_edly_sub_organizations.
+    """
+    full_url = request.build_absolute_uri('?')
+    edly_profile = EdlyUserProfile.objects.filter(user=request.user).first()
+
+    if not edly_profile:
+        return False
+
+    for slug in edly_profile.get_linked_edly_sub_organizations:
```

Comment:
I suggest to get site domain from request and verify that that site is link to user in edly profile.

Reply:
Done